lang/cs/Org.Apache.REEF.Tang/Implementations/Configuration/ConfigurationBuilderImpl.cs

```diff
@@ -85,6 +85,16 @@ protected ConfigurationBuilderImpl(IConfiguration[] confs) : this(new string[0],
         {
         }
 
+        public IConfigurationBuilder AddOptionalConfiguration(IConfiguration conf)
```

Comment:
removed it 

---
In reply to: [84164583](https://github.com/apache/reef/pull/1161#discussion_r84164583) [](ancestors = 84164583)